src/lib/tracer.ts

```diff
@@ -74,6 +74,5 @@ export function createCacheTracer() {
   return {
     get: createCommand("get"),
     set: createCommand("set"),
-    delete: createCommand("delete"),
```

Comment:
I didn't see any use case in the codebase outside of tests for `delete`, and you see we renamed `client.del` to `delete`. Which seems non-standard as both clients implement `del`, and that's what the rate limit middleware wants as well - https://github.com/express-rate-limit/rate-limit-memcached?tab=readme-ov-file#client

So perhaps the `del` part of rate limiter, whatever that was, never quite worked? Then again that middleware is disabled anyway